.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  release:
+    types: [ published ]
```

Comment:
Does this mean we have to create the releases our selves and no longer push a tag?
And will goreleaser update the existing release?

Reply:
You're right. I will fix it.